### PR TITLE
Portfolio

### PR DESCRIPTION
**Pull request by LukasKurpiela**
Good evening all,

I really would appreciate a code-review and feedback to my code changes. Thank you very much in advance!

- Added the History Page per coin, which opens, when coin is clicked in Portfolio overview (second page)
- Added "add button" and "close button" and added "useHistory" in history page and in Form.
- Click on "add button" will open the Form to insert a new buy or sell for a specific coin. This will be included to the history.

Note: Vercel Preview unfortunately NOT AVAILABLE due to issues in deployment (changed folder directory issue).

Not implemented yet / still open:
- News page (third icon in Navbar)
- Calculation of holdings per coin / total Portfolio value
- Deletion of transaction in history and Portfolio overview (+ modal for confirmation)

### CODE REVIEW

File: client/src/Components/Footer.js [hunk 1]
```diff
@@ -37,6 +37,7 @@ const NavWrapper = styled.nav`
   left: 0;
   z-index: 100;
```
Comment (SabMueller) [inline client/src/Components/Footer.js]: You could consider sorting your styles in alphabetical order :)

File: client/src/Components/Footerbutton.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      </NavWrapper>
+    </footer>
+  );
+}
+
+const NavWrapper = styled.nav`
+  padding-bottom: 20px;
+  width: 100%;
+  display: flex;
+  flex-direction: row;
```
Comment (SabMueller) [inline client/src/Components/Footerbutton.js]: You could consider sorting your styles in alphabetical order :) Please check your overall styles, I dont want to post this comment all over.... :D

File: client/src/lib/validation.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+const validateQuantity = (quantity) => quantity > 0;
```
Comment (SabMueller) [inline client/src/lib/validation.js:1]: I really like your validation functions! :)

File: client/src/pages/AddForm.js [hunk 4]
```diff
@@ -0,0 +1,250 @@
...
+      <HeadlineWrapper>
+        <HeadlineName>{name} Add Transaction</HeadlineName>
+        <CloseIcon title="Close" role="img" onClick={navigateToOverview} />
+      </HeadlineWrapper>
+      <BuyOrSell>
+        <label htmlFor="buyOrSell"></label>
+        <input
+          isPrimary
+          type="radio"
+          value="Buy"
```
Comment (SabMueller) [inline client/src/pages/AddForm.js]: I think values are supposed to be in lower case

File: client/src/pages/Portfoliopage.js [hunk 5]
```diff
@@ -1,19 +1,31 @@
 import Portfolio from '../components/Portfolio';
 import Headline from '../components/Headline';
+// import styled from 'styled-components/macro';
```
Comment (SabMueller) [inline client/src/pages/Portfoliopage.js]: if you dont need this import please consider removing it :)

**Review by SabMueller — APPROVED** [review]
Looks good to me! Just a few minor remarks for you to consider, but all in all well done! ✨

**Comment by LukasKurpiela** [comment]
Thank you for the great review and hints Sabrina. I will consider it for the future styling! :)

**Pull request merged**